graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/admin/GqlAdminQuery.java

```diff
@@ -4,19 +4,72 @@
 import graphql.annotations.annotationTypes.GraphQLField;
 import graphql.annotations.annotationTypes.GraphQLName;
 import graphql.annotations.annotationTypes.GraphQLNonNull;
+import org.apache.jackrabbit.util.ISO8601;
+import org.jahia.api.Constants;
 import org.jahia.bin.Jahia;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
 
 /**
  * GraphQL root object for Admin related queries.
  */
 @GraphQLName("adminQuery")
 @GraphQLDescription("Admin queries root")
 public class GqlAdminQuery {
+
+    /**
+     * @deprecated replaced by jahia node
+     */
+    @Deprecated
     @GraphQLField
     @GraphQLName("version")
     @GraphQLNonNull
     @GraphQLDescription("Version of the running Jahia instance")
     public String getProductVersion() {
         return Jahia.getFullProductVersion();
     }
+
+    /**
+     * Get getJahiaVersion
+     *
+     * @return GqlJahiaVersion
+     */
+    @GraphQLField
+    @GraphQLName("jahia")
+    @GraphQLDescription("Version of the running Jahia instance")
+    public GqlJahiaVersion getJahiaVersion()  {
+
+        GqlJahiaVersion gqlJahiaVersion = new GqlJahiaVersion();
+        gqlJahiaVersion.setRelease(Constants.JAHIA_PROJECT_VERSION);
+        gqlJahiaVersion.setBuild(String.valueOf(Jahia.getBuildNumber()));
+        gqlJahiaVersion.setSnapshot(Constants.JAHIA_PROJECT_VERSION.contains("SNAPSHOT"));
+
+        //Formatting buildDate
+        Date date = null;
+        try {
+            date = DateFormat.getDateTimeInstance(DateFormat.LONG, DateFormat.LONG, Locale.ENGLISH).parse(Jahia.getBuildDate());
+            Calendar calendar = Calendar.getInstance();
+            calendar.setTime(date);
+            gqlJahiaVersion.setBuildDate(ISO8601.format(calendar));
+        } catch (ParseException e) {
+            e.printStackTrace();
```

Comment:
add a logger on top of the class like this
public static final Logger logger = LoggerFactory.getLogger(GqlAdminQuery.class);
and then log the exception as warning
logger.warn("Exception while parsing build date",e)

Reply:
Make sense. Im updating now